--- xmule-1.9.4/src/sockets.h.orig	2004-10-30 23:19:11.000000000 +0200
+++ xmule-1.9.4/src/sockets.h	2005-05-17 00:11:04.000000000 +0200
@@ -26,6 +26,7 @@
 
 #include "wintypes.h"
 #include "types.h"
+#include "stdint.h"
 #include "server.h"
 #include "ServerListCtrl.h"
 #include "server.h"
--- xmule-1.9.4/src/ClientList.h.orig	2004-10-30 23:19:11.000000000 +0200
+++ xmule-1.9.4/src/ClientList.h	2005-05-17 00:15:30.000000000 +0200
@@ -24,6 +24,7 @@
 
 #include "updownclient.h"
 #include "types.h"
+#include "stdint.h"
 #include "ListenSocket.h"
 
 #include <map>                              // std::map
