--- xmule-1.4.0/src/xmule.cpp.orig	Wed May 28 14:29:29 2003
+++ xmule-1.4.0/src/xmule.cpp	Wed May 28 14:30:32 2003
@@ -55,16 +55,6 @@
 //	ON_COMMAND(ID_HELP, CWinApp::OnHelp)
 //END_MESSAGE_MAP()
 
-#include <sys/time.h>
-inline long GetTickCount()
-{	
-  struct timeval aika;
-  gettimeofday(&aika,NULL);
-  unsigned long secs=aika.tv_sec*1000;
-  secs+=(aika.tv_usec/1000);
-  return secs;
-}
-
 CxmuleApp::CxmuleApp() {
   splashBmp=new wxBitmap((const char**)About_jpg);
 }
--- xmule-1.4.0/src/updownclient.h.orig	Wed May 28 14:28:23 2003
+++ xmule-1.4.0/src/updownclient.h	Wed May 28 14:29:09 2003
@@ -26,11 +26,21 @@
 #include "SafeFile.h"
 #include "BarShader.h"
 #include "otherfunctions.h"
+#include <sys/time.h>
+
 
 class CPartFile;
 class CKnownFile;
 typedef unsigned char byte;
-extern inline long GetTickCount();
+
+inline long GetTickCount()
+{ 
+  struct timeval aika;
+  gettimeofday(&aika,NULL);
+  unsigned long secs=aika.tv_sec*1000;
+  secs+=(aika.tv_usec/1000);
+  return secs;
+}
 
 
 // uploadstate
