--- xmule-1.12.2/configure.ac.orig	2005-12-03 00:05:56.000000000 +0100
+++ xmule-1.12.2/configure.ac	2005-12-17 10:23:39.608751000 +0100
@@ -1,6 +1,7 @@
 # Process this file with autoconf to produce a configure script.
 AC_PREREQ(2.53)
 AC_INIT([xMule],[1.12.2],[http://www.xmule.ws/])
+AM_INIT_AUTOMAKE(xMule, 1.12.2)
 AC_CONFIG_SRCDIR([src/SharedFileList.h])
 AC_CONFIG_HEADER([config.h])
 CONFIG=Release
